<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolunteeringAppClass</class>
 <widget class="QMainWindow" name="VolunteeringAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VolunteeringApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="descriptionlineEdit"/>
    </item>
    <item>
     <widget class="QListWidget" name="volunteerslistWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Unassigned volunteers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="suitabilitypushButton">
        <property name="text">
         <string>Compute suitability</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="assignpushButton">
        <property name="text">
         <string>Assign to department</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="unassignedlistWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="namelineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emaillineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>List of interests</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="interestslineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addVolunteerpushButton">
        <property name="text">
         <string>Add volunteer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VolunteeringApp.qrc"/>
 </resources>
 <connections/>
</ui>
